docs/index.bs

```diff
@@ -1600,6 +1600,10 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       };
     </pre>
 
+    A <dfn export id="dfn-count-router-condition-result">count router condition result</dfn> is a [=struct=] that consists of:
```

Comment:
nit but the struct might currently be used within this specification, and I guess we do not need to "export" them, unless we want other spec to refer this.

Reply:
Done.